ec2/main.tf

```diff
@@ -47,6 +47,7 @@ resource "aws_iam_role" "ec2_role" {
 }
 
 resource "aws_iam_role_policy_attachment" "ec2_role_policy_attachment" {
-  policy_arn = "arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"
+  for_each   = toset(concat(["arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore"], var.additional_policy_arns))
```

Comment:
Nice. Finally get to use `for_each`

Reply:
Ah sorry, this didn't work because you can't pass things that are created in terraform apply into the foreach so I had to change it and do the attachment in the root.tf in the scripts project. You can only pass in hard coded things which is a bit disappointing. I didn't mean to leave it in, I've taken it out now.

Reply:
Got all excited there for a minute ;)